scripts/tools/zap/generate.py

```diff
@@ -31,7 +31,6 @@ class CmdLineArgs:
     zclFile: str
     templateFile: str
     outputDir: str
-    runBootstrap: bool
```

Comment:
We should probably keep this for the development zap case.

Reply:
Updated